minetest/minetest
# Cleanup shader generation code

Shader generation is a mess. This PR cleans some parts up, including dropping remains of HLSL support which was never actually implemented.

The next part will probably be to drop all the setter-constant-factory-shader nonsense.

## To do

This PR is Ready for Review.

## How to test

Run MT with different video drivers with and without shaders. Nothing should change.
